curvefs/src/tools/list/curvefs_topology_list.h

```diff
@@ -133,36 +137,64 @@ class TopologyListTool
 
     void ShowMetaserverInfo(const MetaserverInfoType& metaserver) const;
 
+    bool BuildClusterMapPools(Json::Value* pools);
+
+    bool BuildClusterMapServers(Json::Value* servers);
+
+    bool BuildJsonValue(Json::Value* value);
+
+    bool GetClusterTree(Json::Value* cluster, const std::string& clusterId);
+
+    bool GetPoolTree(Json::Value* pool, uint64_t poolId);
+
+    bool GetZoneTree(Json::Value* zone, uint64_t zoneId);
+
+    bool GetServerTree(Json::Value* server, uint64_t serverId);
+
+    bool GetMetaserverTree(Json::Value* metaserver, uint64_t metaserverId);
```

Comment:
keep a class small, so this format logic should not be put in here.
condier define tree builder / json builder into single class

Reply:
> keep a class small, so this format logic should not be put in here. condier define tree builder / json builder into single class

fix